### PR TITLE
Remove call to helmclient.EnsureTillerInstalled on boot

### PR DESCRIPTION
**Pull request by rossf7**
Fixes https://github.com/giantswarm/giantswarm/issues/8068

The call to EnsureTillerInstalled on boot was added in https://github.com/giantswarm/draughtsman/pull/74 as a workaround to upgrade tiller because control plane clusters did not have any chartconfig CRs.

Now cluster-operator is deployed with VOO all control planes have chart CRs and chart-operator can take care of managing tiller.

This removes some dumb race conditions when draughtsman and chart-operator fight over the tiller deployment.

### CODE REVIEW

**Comment by rossf7** [comment]
Tested on `geckon`.

**Review by tomahawk28 — APPROVED** [review]
LGTM

File: Gopkg.toml [hunk 1]
```diff
@@ -72,34 +68,12 @@
...
-  version = "v1.16.3"
-
-[[override]]
-  name = "github.com/russross/blackfriday"
-  version = "v1.5.2"
-
-[[override]]
-  name = "github.com/ghodss/yaml"
-  branch = "master"
+  source = "gopkg.in/fsnotify/fsnotify.v1"
```
Comment (sslavic) [inline Gopkg.toml]: Seems this fsnotify override can be removed as well.

**Review by sslavic — APPROVED** [review]
draughtsman on a diet! LGTM

File: Gopkg.toml (hunk 1 above)
Comment (rossf7) [reply Gopkg.toml]: Thanks! It's gone.

**Pull request merged**